Modules/posixmodule.c

```diff
@@ -15301,6 +15304,9 @@ all_ins(PyObject *m)
 
 #if defined(__APPLE__)
     if (PyModule_AddIntConstant(m, "_COPYFILE_DATA", COPYFILE_DATA)) return -1;
+    if (PyModule_AddIntMacro(m, O_EVTONLY)) return -1;
+    if (PyModule_AddIntMacro(m, O_FSYNC)) return -1;
+    if (PyModule_AddIntMacro(m, O_SYMLINK)) return -1;
```

Comment:
Please use the same pattern as for the other O_* constants, outside of the ``#if __APPLE__`` block.